dlrover/python/elastic_agent/master_client.py

```diff
@@ -310,7 +310,8 @@ def num_nodes_waiting(self, rdzv_name):
 
     def join_rendezvous(self, node_rank, local_world_size, rdzv_name=""):
         request = grpc.JoinRendezvousRequest(
-            node_id=node_rank,
+            node_id=self._node_id,
```

Comment:
Is the 'node-id' equals to the pod name's suffix?

Reply:
Yes.